trpcultivate_phenotypes/src/Service/TripalCultivatePhenotypesTraitsService.php

```diff
@@ -193,6 +215,12 @@ public function setTraitGenus(string $genus) {
   public function insertTrait(array $trait, ?string $schema = NULL) {
     // Configuration settings of the genus.
     $genus_config = $this->config;
+
+    if ($schema) {
+      $this->cvterm_buddy->connection->setSchemaName($schema);
+      $this->dbxref_buddy->connection->setSchemaName($schema);
+    }
```

Comment:
Would it be possible to get code coverage for this block, or is it also related to issues with mocking ChadoBuddy?